### PR TITLE
Simplify projects to SDK style

### PR DESCRIPTION
**Pull request by wcoder**
### Description

Simplify platform projects to SDK style with compatibility with current apps.

### Issues

- Preparation for #450

### API Changes
 
 None

### Platforms Affected

- Core (all platforms)
- iOS
- Android

### Behavioral/Visual Changes

None

### Before/After Screenshots

Not applicable

### PR Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/Softeq/XToolkit.WhiteLabel/blob/master/.github/CONTRIBUTING.md) document
- [x] My code follows the [code styles](https://github.com/Softeq/dotnet-guidelines)
- [x] Targets the correct branch
- [x] Tests are passing (or failures are unrelated)

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -4,6 +4,11 @@
   <PropertyGroup>
     <LangVersion>8.0</LangVersion>
     <Nullable>enable</Nullable>
+
+    <!-- Assembly Info -->
+    <Product>XToolkit</Product>
+    <Company>Softeq Development Corporation</Company>
+    <Copyright>Copyright © 2022 Softeq Development Corporation</Copyright>
```
Comment (nkrisko) [inline Directory.Build.props:11]: 2023?

File: Softeq.XToolkit.Bindings.Droid/Softeq.XToolkit.Bindings.Droid.csproj [hunk 2]
```diff
@@ -1,84 +1,33 @@
...
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
     <AndroidLinkMode>SdkOnly</AndroidLinkMode>
   </PropertyGroup>
+
   <ItemGroup>
-    <PackageReference Include="Xamarin.AndroidX.RecyclerView" Version="1.2.1.7" />
+    <None Include="Resources\*.cs" />
```
Comment (nkrisko) [inline Softeq.XToolkit.Bindings.Droid/Softeq.XToolkit.Bindings.Droid.csproj:18]: looks like some line(s) is unnecessary

File: Softeq.XToolkit.Common.iOS.Tests/Softeq.XToolkit.Common.iOS.Tests.csproj [hunk 3]
```diff
@@ -1,132 +1,61 @@
...
-    <Reference Include="System.Core" />
-    <Reference Include="Xamarin.iOS" />
-  </ItemGroup>
-  <ItemGroup>
-    <InterfaceDefinition Include="LaunchScreen.storyboard" />
-  </ItemGroup>
+
   <ItemGroup>
     <None Include="Info.plist" />
     <None Include="Entitlements.plist" />
```
Comment (nkrisko) [inline Softeq.XToolkit.Common.iOS.Tests/Softeq.XToolkit.Common.iOS.Tests.csproj:49]: is it really need?

File: Softeq.XToolkit.WhiteLabel.Droid/Softeq.XToolkit.WhiteLabel.Droid.csproj [hunk 4]
```diff
@@ -1,145 +1,38 @@
...
     <AndroidLinkMode>SdkOnly</AndroidLinkMode>
   </PropertyGroup>
+
   <ItemGroup>
-    <PackageReference Include="Xamarin.Essentials" Version="1.7.3" />
-    <PackageReference Include="Xamarin.FFImageLoading" Version="2.4.11.982" />
-    <PackageReference Include="Xamarin.AndroidX.AppCompat" Version="1.5.0" />
-    <PackageReference Include="Xamarin.Google.Android.Material" Version="1.6.1.1" />
+    <None Include="Resources\*.cs" />
+    <Compile Remove="Resources\*.cs" />
```
Comment (nkrisko) [inline Softeq.XToolkit.WhiteLabel.Droid/Softeq.XToolkit.WhiteLabel.Droid.csproj:19]: is it really need?

File: Softeq.XToolkit.WhiteLabel.Essentials.Droid/Softeq.XToolkit.WhiteLabel.Essentials.Droid.csproj [hunk 5]
```diff
@@ -1,95 +1,37 @@
...
+    <AndroidLinkMode>None</AndroidLinkMode>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <AndroidLinkMode>SdkOnly</AndroidLinkMode>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="Resources\*.cs" />
+    <Compile Remove="Resources\*.cs" />
```
Comment (nkrisko) [inline Softeq.XToolkit.WhiteLabel.Essentials.Droid/Softeq.XToolkit.WhiteLabel.Essentials.Droid.csproj:19]: is it really need?

File: samples/Playground/Playground.iOS/Playground.iOS.csproj [hunk 6]
```diff
@@ -1,382 +1,81 @@
...
-    <InterfaceDefinition Include="Views\MovieTableViewCell.xib" />
-    <InterfaceDefinition Include="ViewControllers\Controls\PhotoBrowserStoryboard.storyboard" />
-    <InterfaceDefinition Include="ViewControllers\Components\ConnectivityPageStoryboard.storyboard" />
-    <InterfaceDefinition Include="ViewControllers\Components\GesturesPageStoryboard.storyboard" />
-    <InterfaceDefinition Include="ViewControllers\Collections\CompositionalLayoutPageStoryboard.storyboard" />
-    <InterfaceDefinition Include="Views\Collections\DummyCell.xib" />
-  </ItemGroup>
+
   <ItemGroup>
     <None Include="Info.plist" />
```
Comment (nkrisko) [inline samples/Playground/Playground.iOS/Playground.iOS.csproj:48]: is it really need?

Comment (wcoder) [reply samples/Playground/Playground.iOS/Playground.iOS.csproj:48]: yes

File: Softeq.XToolkit.Common.iOS.Tests/Softeq.XToolkit.Common.iOS.Tests.csproj (hunk 3 above)
Comment (wcoder) [reply Softeq.XToolkit.Common.iOS.Tests/Softeq.XToolkit.Common.iOS.Tests.csproj:49]: yes

**Review by vadimpylsky — APPROVED** [review]

**Pull request merged**